zstacklib/zstacklib/utils/test_pci_compat.py

```
@@ -0,0 +1,273 @@
+        self.assertIsInstance(error, str)
+
+    def test_devices_separated_by_blank_line(self):
+        """Multiple devices should be separated by a blank line (\\n\\n).
+
+        This matches the lspci output format that _parse_lspci_output expects.
+        """
+        self.sysfs.add_device("0000:3b:00.0", vendor="0x10de", device="0x2684")
+        self.sysfs.add_device("0000:42:00.0", vendor="0x1002", device="0x7340")
+        self.sysfs.add_device("0000:00:02.0", vendor="0x8086", device="0x29c0")
+
+        returncode, output, error = get_pci_device_ids(
```

Ruff (0.15.2)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)